### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by wayofthefuture** [nodequery/nq-agent#17]
What is the status of NodeQuery.com? Are things still running or has this project been deprecated?

**Comment by neobie** [comment]
Yes I am wondering too. The project seems no more update 2 years ago.
But, the domain is still renewing.

**Comment by wayofthefuture** [comment]
I think it's no longer being updated... I found something called "pm2" which works nicely.

**Comment by iamkingsleyf** [comment]
i have removed from all my servers

**Comment by pcnate** [comment]
One of the contributers, @tvjames hasn't posted on his blog since November 2015.... Has made commits in github for other projects but not very much... This is a shame, it still works very nice. I had gotten excited about the enterprise piece until I realized how neglected the project has been....

**Comment by tvjames** [comment]
@pcnate I'll just add that I have nothing to do with this project. I just fixed a bug a long time ago. I also dont use it any more.

**Comment by pcnate** [comment]
Thanks for the info

**Comment by LeBaux** [comment]
2019, seems like today the service might have gone down for the good.
It is a damn shame because I liked it for simple monitoring a lot.
So what are you guys using instead?

**Comment by iamkingsleyf** [comment]
I use https://nixstats.com/

**Comment by jaden** [comment]
> So what are you guys using instead?

It was down again for a while today. I've switched to HetrixTools.

**Comment by LeBaux** [comment]
At first glance, HetrixTools pack much more in the free tier.
On the other hand, NixStats looks more customizable.

**Comment by chocolatkey** [comment]
I noticed that the copyright date on the site has been updated from 2014 -> 2019... Maybe something new?

**Comment by pcnate** [comment]
i expect that is auto generated

**Comment by agungyuliaji** [comment]
I wonder if nodequery is owned and maintained by one person? maybe that person died? and no one has access to this project? the reason i think like that because:
- this makes no sense, public open beta for more than 7 years
- it stopped completely, no reply to email or github and social media
